_data/banking.yml

```
@@ -1055,38 +1321,50 @@ websites:
     email_address: servizio.clienti@posteitaliane.it
     img: posteitaliane.png
     lang: it
+    regions:
+      - it
 
   - name: Rabobank
     url: https://www.rabobank.nl/
     img: rabobank.png
     tfa:
       - hardware
     doc: https://www.rabobank.nl/bedrijven/betalen/zelf-betalingen-doen/rabo-scanner/
+    regions:
+      - nl
 
   - name: Raiffeisen Bank CZ
     url: https://www.rb.cz/
     img: raiffeisenbank_cz.png
     doc: https://www.rb.cz/firemni-finance/podnikatele-a-male-firmy/prime-bankovnictvi/sluzby-pro-firemni-ucty/bezpecnost-internetoveho-bankovnictvi/
     tfa:
       - hardware
+    regions:
+      - cz
 
   - name: RBC Royal Bank
     url: https://www.rbcroyalbank.com
     facebook: rbc
     img: rbc.png
+    regions:
+      - gb
```

```suggestion
      - ca
```
RBC (Royal Bank of Canada) is a bank in Canada. It's insured by CDIC.